### ISSUE TITLE
Users part of Plex Home

### ISSUE THREAD

**Issue by cyberweasel** [aunefyren/wrapperr#75]
I turned off Plex Auth and just have the field to type in a plex user name.  If I type a full plex user name it works.  If I type a Home user account that does not have a full plex account, I get a "User validation error."  I think this used to work in older versions.

**Comment by vakarian92** [comment]
I'm having the same. I checked and Wrapperr can access Tautulli and all the users but then in the logs I see this:

```
2023/11/27 00:55:36 Retrieved Wrapperr functions. - Origin: MYIP 
[GIN] 2023/11/27 - 00:55:36 | 200 |     689.331µs |   MYIP | POST     "/api/get/functions"
2023/11/27 00:55:36 New wrap request.
2023/11/27 00:55:36 Checking Tautulli server 'Tautulli'.
2023/11/27 00:55:36 Getting users from Tautulli server 'Tautulli'.
2023/11/27 00:55:36 At this point the user should have been verified, but username, email or ID is empty.
[GIN] 2023/11/27 - 00:55:36 | 500 |   41.800512ms |   MYIP | POST     "/api/get/statistics"
```
Note: I changed my IP with MYIP

I also tried to use a random string as username and I get this error. So I think the username is recognized.

```
2023/11/27 00:58:14 Checking Tautulli server 'Tautulli'.
2023/11/27 00:58:14 Getting users from Tautulli server 'Tautulli'.
2023/11/27 00:58:14 Could not find any user that matched the given Plex Identity: 'afdwf'.
2023/11/27 00:58:14 Failed to find user in Tautulli.
```

**Comment by aunefyren** [comment]
https://github.com/aunefyren/wrapperr/blob/80720039e88e941fafc7556cfec34a505538e949/routes/both.go#L285

Mmm, yes. I throw an error if e-mail is not found. Figured it was a good validation, but managed users probably do not have e-mail addresses attached.

**Comment by SerinitySW** [comment]
This is happening on my end as well. Interestingly in home users do properly show up in the server-wide statistics. But I receive the error mentioned above when trying to parse the user's wrapped.

**Comment by aunefyren** [comment]
> This is happening on my end as well. Interestingly in home users do properly show up in the server-wide statistics. But I receive the error mentioned above when trying to parse the user's wrapped.

Yes, that is true. That is because nothing is really wrong with the statistics themselves, the validation is just too strict toward the statistics request. 😅

**Comment by dpeet** [comment]
Just came here with the same issue 😅 Thankful for your quick fix!  I see you've got the relevant code committed, do you have thoughts on when it will get released?  I've got the project set up in docker.

**Comment by aunefyren** [comment]
> Just came here with the same issue 😅 Thankful for your quick fix! I see you've got the relevant code committed, do you have thoughts on when it will get released? I've got the project set up in docker.

No worries, should be out today. Will close the issue on release.

**Comment by aunefyren** [comment]
Fixed in v3.2.2.
